docs/reference/mapping/params/ignore-malformed.asciidoc

```diff
@@ -46,15 +46,23 @@ PUT my_index/_doc/2
 <1> This document will have the `text` field indexed, but not the `number_one` field.
 <2> This document will be rejected because `number_two` does not allow malformed values.
 
+The `ignore_malformed` setting is currently supported by the following <<mapping-types,mapping types>>:
+* <<number>>::         `long`, `integer`, `short`, `byte`, `double`, `float`, `half_float`, `scaled_float`
+* <<date>>::           `date`
+* <<date_nanos>>::     `date_nanos`
+* <<geo-point>>::     `geo_point` for lat/lon points
+* <<geo-shape>>::     `geo_shape` for complex shapes like polygons
+* <<ip>>::            `ip` for IPv4 and IPv6 addresses
```

Comment:
This doesn't render properly in the output:

<img width="732" alt="def-output" src="https://user-images.githubusercontent.com/40268737/67112855-29d1ef00-f1a6-11e9-9a62-20bc4c1cc254.png">

If you want a bulleted list, you can try this instead (note the added new line):

```
The `ignore_malformed` setting is currently supported by the following <<mapping-types,mapping types>>:

* <<number>>: `long`, `integer`, `short`, `byte`, `double`, `float`, `half_float`, `scaled_float`
* <<date>>: `date`
* <<date_nanos>>: `date_nanos`
* <<geo-point>>: `geo_point` for lat/lon points
* <<geo-shape>>: `geo_shape` for complex shapes like polygons
* <<ip>>: `ip` for IPv4 and IPv6 addresses
```